.gcal.php

```diff
@@ -0,0 +1,85 @@
+<?php
+require_once 'vendor/autoload.php';
+require_once '.functions.php';
+
+putenv('GOOGLE_APPLICATION_CREDENTIALS=gcal_creds.json');
+
+Class GoogleCalendar {
+
+    private $service;
+    private $calendar_id = 'rpiambulance.com_uuc6tmlmf4ct8qeps19oj1i69g@group.calendar.google.com';
+
+    function __construct() {
+      $client = new Google_Client();
+      $client->useApplicationDefaultCredentials();
+      $client->setApplicationName('RPIA Calendar');
+      $client->addScope(Google_Service_Calendar::CALENDAR);
+      $client->addScope(Google_Service_Calendar::CALENDAR_READONLY);
```

Comment:
Is the `READONLY` scope necessary?

Reply:
CALENDAR is for editing, CALENDAR_READONLY is for reading from the calendar.